beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1835,6 +1835,16 @@ pub static BLOCK_PRODUCTION_BLOBS_VERIFICATION_TIMES: LazyLock<Result<Histogram>
     },
 );
 
+/*
+ * Availability related metrics
+ */
+pub static BLOCK_AVAILABILITY_DELAY: LazyLock<Result<IntGauge>> = LazyLock::new(|| {
+    try_create_int_gauge(
+        "block_availability_delay",
+        "Duration between start of the slot and the time at which all components of the block are available.",
+    )
+});
```

Comment:
Actually reviewed current unstable and we have `BEACON_BLOCK_DELAY_AVAILABLE_SLOT_START` which tracks exactly this but with more precision. Ok to delete `BLOCK_AVAILABILITY_DELAY`